ta/pkcs11/src/pkcs11_token.c

```diff
@@ -661,16 +674,14 @@ static void close_ck_session(struct pkcs11_session *session)
 {
 	release_active_processing(session);
 
-	/* No need to put object handles, the whole database is destroyed */
+	release_session_find_obj_context(session);
+	/* Release all session objects */
```

Comment:
prefer an empty line above. Maybe:
```c
	release_active_processing(session);
	release_session_find_obj_context(session);

	/* Release all session objects */
	while (!LIST_EMPTY(&session->object_list))
		destroy_object(session,
			       LIST_FIRST(&session->object_list), true);

	...

Reply:
Sure. Was a bit unsure last round what was wanted -- now this is clear.